substrate/frame/staking-async/ah-client/src/lib.rs

```diff
@@ -129,6 +154,22 @@ impl<T: Config + pallet_session::Config + pallet_session::historical::Config> Se
 	fn report_offence(offender: Self::ValidatorId, severity: OffenceSeverity) {
 		pallet_session::Pallet::<T>::report_offence(offender, severity)
 	}
+
+	fn set_keys(account: &Self::AccountId, keys: Self::Keys) -> DispatchResult {
+		pallet_session::Pallet::<T>::do_set_keys(account, keys)
+	}
+
+	fn purge_keys(account: &Self::AccountId) -> DispatchResult {
```

Comment:
I feel this should have been implemented by Session pallet, right? But given its already here, its probably fine to leave it as it is.